src/renderer/js/redux/reducers/app.js

```diff
@@ -164,7 +187,7 @@ reducers[types.VOLUME_CHANGED] = function(state, action) {
   });
 };
 
-export default function reducer(state = defaultState, action) {
+export default function reducer(state: appState = defaultState, action: any) {
```

Comment:
We should have an `action` type

Reply:
completely agreed, I'm trying to figure that out now

There is some cool stuff we can do with flow and actions, but I also don't want to introduce too much unnecessary boilerplate if its not going to practically help us catch errors